esim_research_migration_details.inc

```diff
@@ -0,0 +1,104 @@
+<?php
+// research migration display completed proposals
+function esim_research_migration_completed_proposals_all()
+{
+	$output = "";
+	$query = db_select('research_migration_proposal');
+	$query->fields('research_migration_proposal');
+	$query->condition('approval_status', 3);
+	$query->orderBy('actual_completion_date', 'DESC');
+	//$query->condition('is_completed', 1);
+	$result = $query->execute();
+	
+	//var_dump($research_migration_abstract);die;
+	if ($result->rowCount() == 0)
+	{
+		$output .= "Work has been completed for the following research migrations. We welcome your contributions." ."<hr>";
```

Comment:
Rephrase the message as "Currently, there are no submissions in this section. Click here to propose a Research Migration Project."